admin/class-fastcgi-purger.php

```diff
@@ -131,7 +131,7 @@ public function custom_purge_urls() {
 			case 'get_request':
 				// Go to default case.
 			default:
-				$_url_purge_base = $parse['scheme'] . '://' . $parse['host'] . '/purge';
+				$_url_purge_base = $this->purge_base_url() . '/purge';
```

Comment:
I think you need to remove `/purge` because `purge_base_url()` will return URL base with suffix.

`$_url_purge_base = $this->purge_base_url();`

Reply:
Ah, ofcourse, missed that in the PR :)